src/equations/compressible_euler_1d.jl

```diff
@@ -825,8 +825,8 @@ Compactly summarized:
   Numerical methods for conservation laws and related equations.
   [Link](https://metaphor.ethz.ch/x/2019/hs/401-4671-00L/literature/mishra_hyperbolic_pdes.pdf)
 """
-function flux_hlle(u_ll, u_rr, orientation::Integer,
-                   equations::CompressibleEulerEquations1D)
+function min_max_speed_einfeldt(u_ll, u_rr, orientation::Integer,
```

Comment:
```suggestion
@inline function min_max_speed_einfeldt(u_ll, u_rr, orientation::Integer,
```
I guess all these estimates should be `@inline`ed to improve performance?